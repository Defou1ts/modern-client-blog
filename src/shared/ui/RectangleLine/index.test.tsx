import React from 'react';

import renderer from 'react-test-renderer';

import { RectangleLine } from './index';

describe('RectangleLine', () => {
	it('RectangleLine snapshot', () => {
		const component = renderer.create(<RectangleLine />);

		const tree = component.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
