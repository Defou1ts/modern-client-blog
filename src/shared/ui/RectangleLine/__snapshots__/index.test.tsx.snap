// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RectangleLine RectangleLine snapshot 1`] = `
<div
  className="rectangles"
>
  <div
    className="purpleRectangle"
  />
  <div
    className="yellowRectangle"
  />
</div>
`;
